### PR TITLE
Symorton/issuance callback network layer

### PR DESCRIPTION
**Pull request by symorton**
**Problem:**
We need a way for issuers to know the outcome of their issuance. 

**Solution:**
Issuance Completion Response Network Call.
This is PR 1/2. Just added the data models and changes to the network layer. Will submit another PR with changes to VCServices layer next.

**Validation:**
Please include here how it was verified that the PR works as intended.

**Type of change:**
- [X] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [X] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: VCNetworking/VCNetworking/IssuanceNetworkCalls.swift [hunk 1]
```diff
@@ -48,4 +48,20 @@ public class IssuanceNetworkCalls: IssuanceNetworking {
             }
         }
     }
+    
+    public func sendCompletionResponse(usingUrl url: String,
+                                       withBody body: IssuanceCompletionResponse) -> Promise<String?> {
+        do {
+            var operation = try PostIssuanceCompletionResponseOperation(usingUrl: url,
+                                                              withBody: body,
```
Comment (dangodb) [inline VCNetworking/VCNetworking/IssuanceNetworkCalls.swift:56]: indentation

File: VCNetworking/VCNetworking/decoders/BasicServiceResponseDecoder.swift [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import Foundation
+
+public struct BasicServiceResponseDecoder: Decoding {
+    public func decode(data: Data) throws -> String? {
+        return nil
```
Comment (dangodb) [inline VCNetworking/VCNetworking/decoders/BasicServiceResponseDecoder.swift:10]: Does it ignore the body? In that case I would probably name it something like NilServiceResponseDecoder? Or can we make the place where it's used an optional and not pass one?

File: VCNetworking/VCNetworking/operations/post/PostPresentationResponseOperation.swift [hunk 3]
```diff
@@ -11,7 +11,7 @@ class PostPresentationResponseOperation: InternalPostNetworkOperation {
     typealias RequestBody = PresentationResponse
     typealias ResponseBody = String?
     
-    let decoder = PresentationServiceResponseDecoder()
+    let decoder = BasicServiceResponseDecoder()
```
Comment (dangodb) [inline VCNetworking/VCNetworking/operations/post/PostPresentationResponseOperation.swift:14]: Expected?

**Review by dangodb — APPROVED** [review]

File: VCNetworking/VCNetworking/operations/post/PostPresentationResponseOperation.swift (hunk 3 above)
Comment (symorton) [reply VCNetworking/VCNetworking/operations/post/PostPresentationResponseOperation.swift:14]: yep

**Pull request merged**